breathe/renderer/sphinxrenderer.py

```diff
@@ -687,7 +687,10 @@ def content(contentnode):
         assert isinstance(sig, addnodes.desc_signature)
 
         # Insert the member name for use in Sphinx-generated table of contents.
-        member_name = node.get_name()
+        if node.node_type == "compounddef":
+            member_name = node.compoundname
+        else:
+            member_name = node.name
```

Comment:
Do we know what's changed here?

Reply:
My guess is it was simply overlooked in #959.